### DISCUSSION TITLE
Mod sharing

### DISCUSSION THREAD

**Discussion by Epicpkmn11** [Epicpkmn11/WordleDS#12] [Mod Sharing]
Made a mod for Wordle DS and want to share it with others? Upload it here alongside a description of the mod and how to credit you!

Mods uploaded here will be added to the [mods branch](https://github.com/Epicpkmn11/WordleDS/tree/mods) of this repo for easy downloading from Universal-Updater and from the [mod list page](https://github.com/Epicpkmn11/WordleDS/blob/mods/mods.md). By uploading here you agree that your mod can be added to the repo under a CC0 license.

Template:
```md
**Mod name**: 
**Author**: 
**Description**: 
**Categories**: 
**Mod ZIP**: 
**Icon**: 
```
(Upload files by drag droping the file onto the GitHub editor, icon should be 48×48 resolution)

**Comment by Epicpkmn11** [comment]
**EXAMPLE, PLEASE PROVIDE THE SAME INFORMATION WHEN SHARING YOUR MODS**

**Mod name**: Wordle (ES)
**Author**: [Pk11](https://github.com/Epicpkmn11)
**Description**: Wordle DS translated to Spanish, based on Daniel Rodriguez's [Wordle (ES)](https://wordle.danielfrg.com).
**Categories**: Translation, Spanish
**Mod ZIP**: [Wordle (ES).zip](https://github.com/Epicpkmn11/WordleDS/files/8550781/Wordle.ES.zip)
**Icon**: ![wordle-es](https://user-images.githubusercontent.com/41608708/165007128-4d317fe4-f72d-47a2-9873-bed4f98cdc92.png)

**Comment by ItsSiem** [comment]
**Mod name**: Numberle DS
**Author**: [ItsSiem](https://www.github.com/itssiem)
**Description**: Wordle DS but with numbers instead of letters.
**Categories**: Alternate Gamemode, Numbers
**Mod ZIP**: [Numberle DS.zip](https://github.com/ItsSiem/NumberleDS/releases/latest/download/Numberle.DS.zip)
**Icon**: ![icon](https://user-images.githubusercontent.com/47294812/165478691-7c430081-e4ae-4b12-b343-57573c18e270.png)

**Reply by Epicpkmn11** [reply]
Added! b7e708eae89a164bf4b13d721b3011f53a70a730

> Numberle DS 312 6/6*
> 
> ⬜🟩⬜⬜⬜
> ⬜🟩🟨⬜🟨
> 🟨🟩🟩⬜⬜
> ⬜🟩🟩⬜🟩
> 🟩🟩🟩⬜🟩
> 🟩🟩🟩🟩🟩

**Reply by ItsSiem** [reply]
Hey, I've updated the mod and I have changed the link to the zip. I hope that's okay!

**Comment by basddeg** [comment]
Mod name: Wordle (IT)
Author: [brownboi2](https://github.com/brownboi2)
Description: An Italian translation mod for WordleDS by brownboi2, a non-italian using Google Translate and an Italian word bank.
Categories: Translation, Italian
Mod ZIP: [Wordle (IT).zip](https://github.com/Epicpkmn11/WordleDS/files/8578789/Wordle.IT.zip)
Icon: ![165007128-4d317fe4-f72d-47a2-9873-bed4f98cdc92](https://user-images.githubusercontent.com/89373223/165657756-2f7f84a4-5ef9-4679-9702-655b3550004d.png)

**Reply by Epicpkmn11** [reply]
Added in 86e48e2a1118666f17aa06d21df867f8b7520028!

FYI I made a couple tweaks to the zip:
- Removed `stats.json` since that got left in
- Changed ᵃ to ª for the numberSuffix, they may look the same depending on your font but the first is "MODIFIER LETTER SMALL A" where the second is "FEMININE ORDINAL INDICATOR" and only the latter is in the `main.nftr` being used

> Wordle (IT) DS 312 4/6* 
> 
> ⬜⬜⬜⬜⬜ 
> ⬜⬜⬜⬜🟩 
> 🟩🟨⬜⬜🟩 
> 🟩🟩🟩🟩🟩

**Comment by NightScript370** [comment]
**Mod Name:** Jazle DS
**Author:** [oradwell](https://github.com/oradwell)
**Description:** A port of the game to realize that NPM should have quality control. The classic game of guessing the JavaScript library, now on DS! Just be glad it's not a framework.
**Categories:** Alternative Gamemode, Challenging, JavaScript,
Mod ZIP: [Mod ZIP.zip](https://github.com/NightScript370/WordleDS-Injector/suites/9039719165/artifacts/417524372)
Icon: ![icon](https://jazle.quest/logo128.png)

**Comment by LedyBacer** [comment]
**Mod name**: Wordle DS (RU)
**Author**: [Bacer](https://github.com/LedyBacer)
**Description**: Wordle DS translated to Russian, words from [KarKarbI4](https://github.com/KarKarbI4)/[wordle_ru](https://github.com/KarKarbI4/wordle_ru)
**Categories**: Translation, Russian
**Mod ZIP**: [Wordle DS (RU).zip](https://github.com/Epicpkmn11/WordleDS/files/10461365/Wordle.DS.RU.zip)
**Icon**: ![Wordle (RU)](https://user-images.githubusercontent.com/19615437/213565963-4a93bca7-3bd4-47ae-bbd3-18b94c76a3d8.png)

**Comment by RainbowTH1** [comment]
**Mod name**: Le Mot DS (FR)
**Author**: [RainbowTH](https://github.com/RainbowTHOfficial)
**Description**: Wordle DS translated to French by RainbowTH, (Not French) based on @louanben's [Wordle (FR)](https://wordle.louan.me/) and using Google Translate.
**Categories**: Translation, French
**Mod ZIP**: [Le Mot DS (FR).zip](https://github.com/Epicpkmn11/WordleDS/files/10529890/Le.Mot.DS.FR.zip)
**Icon**: ![icon](https://user-images.githubusercontent.com/61111446/215296945-80851ba4-7f99-44e6-a51d-d0a169747e60.png)

**Comment by DonutEatMyDonuts** [comment]
Mod Name: Dark Mod
Author: [DonutEatMyDonuts](https://github.com/DonutEatMyDonuts)
Description: WIP Dark Mode for Wordle DS
Categories: Graphics
Mod ZIP: [darkmod.zip](https://github.com/user-attachments/files/31074008/darkmod.zip)
Icon: icon.png
<img width="32" height="32" alt="icon 0" src="https://github.com/user-attachments/assets/03933a20-045c-43a4-b738-a59d5fb0d7b6" />
